src/igvfd/schemas/changelogs/in_vitro_system.md

```diff
@@ -1,5 +1,13 @@
 ## Changelog for *`in_vitro_system.json`*
 
+### Schema version 27
+
+* Remove `cell_fate_change_treatments`.
+
+### Schema version 26
+
+* Restrict `pooled cell specimen` to specify an additional `classifications`.
```

Comment:
Was this a rebase error? Just a bit confused here.

Reply:
The changelog for Schema version 26 was never added so I added it here

Reply:
I wasn't sure how to word it, so that is my attempt.